app/gateway/2.8.x/admin-api/developers/reference.md

```diff
@@ -0,0 +1,1417 @@
+
+```
+HTTP/1.1 204 No Content
+```
+
+
+## Applications
+
+### List all applications for a developer
+
+**Endpoint**
+
+<div class="endpoint get">/developers/{DEVELOPER_EMAIL|DEVELOPER_ID}/applications</div>
+
+Attribute                         | Description
+---------:                        | --------   
+`{DEVELOPER_EMAIL|DEVELOPER_ID}`  | The email or UUID of a developer.
+
+
+**Response**
+```
+HTTP/1.1 200 OK
+```
+
+```json
+{
+    "data": [
+        {
+            "consumer": {
+                "id": "6e31bf1e-dbcb-4a31-bac9-a192fa24f088"
+            },
+            "created_at": 1644963627,
+            "developer": {
+                "id": "5f60930a-ad12-4303-ac5a-59d121ad4942"
+            },
+            "id": "5ff48aaf-3951-4c99-a636-3b682081705c",
+            "name": "example_app",
+            "redirect_uri": "http://mockbin.org",
+            "updated_at": 1644963627
+        },
+        {
+            "consumer": {
+                "id": "2390fd02-bbcd-48f1-b32f-89c262fa68a8"
+            },
+            "created_at": 1644963657,
+            "developer": {
+                "id": "5f60930a-ad12-4303-ac5a-59d121ad4942"
+            },
+            "id": "94e0f633-e8fd-4647-a0cd-4c3015ff2722",
+            "name": "example_app2",
+            "redirect_uri": "http://mockbin.org",
+            "updated_at": 1644963657
+        }
+    ],
+    "next": null,
+    "total": 2
+}
+
+```
+
+### Inspect an application
+
+**Endpoint**
+
+<div class="endpoint get">/developers/{DEVELOPER_EMAIL|DEVELOPER_ID}/applications/{APPLICATION_ID}</div>
+
+Attribute                         | Description
+---------:                        | --------   
+`{DEVELOPER_EMAIL|DEVELOPER_ID}`  | The email or UUID of a developer.
+`{APPLICATION_ID}`  | The application UUID.
+
+**Response**
+```
+HTTP/1.1 200 OK
+```
+
+```json
+{
+    "consumer": {
+        "id": "d5ea9628-f359-4b96-9f76-61562aaf4300"
+    },
+    "created_at": 1644965555,
+    "custom_id": "billing",
+    "developer": {
+        "id": "5f60930a-ad12-4303-ac5a-59d121ad4942"
+    },
+    "id": "ca0d62bd-4616-4b87-b947-43e33e5418f0",
+    "name": "testapp",
+    "redirect_uri": "http://mockbin.org",
+    "updated_at": 1644965555
+}
+```
+
+
+### Create an application
+
+**Endpoint**
+
+<div class="endpoint post">/developers/{DEVELOPER_EMAIL|DEVELOPER_ID}/applications</div>
+
+
+Attribute                         | Description
+---------:                        | --------   
+`{DEVELOPER_EMAIL|DEVELOPER_ID}`  | The email or UUID of a developer.
+
+**Request body**
+
+Attribute                    | Description
+---------:                   | --------   
+`name`<br>*required*         | A name for the application.
+`redirect_uri`<br>*required* | The application's URI.
+`custom_id`                  | A custom description for the application. <br>*Not actually sure what this does*
+`developer` | *Can't tell what this does, kind of looks like it didn't do anything; probably don't want to document?*
```

Comment:
As the text says - couldn't figure out what `developer` does. I was able to set _something_ but it didn't seem like it actually did anything?

Reply:
The `developer` attribute should be read-only here. The developer will be assigned from the UUID or email in the path of the POST request and appears to be ignored if you use it in the body.